dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -567,6 +568,30 @@
 		i2c = <&i2c3>;
 		status = "disabled";
 	};
+
+	sai1_a: sai1@40015404 {
+		compatible = "st,stm32-sai";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x40015404 0x20>;
+		clocks = <&rcc STM32_CLOCK(APB2, 21U)>,
```

Comment:
For consistency in STM32 SoCs DTSI files (be kind my @mathieuchopstm, see https://github.com/zephyrproject-rtos/zephyr/pull/93912):
```suggestion
		clocks = <&rcc STM32_CLOCK(APB2, 21)>,
```
Ditto for `sai1_b` below.